aten/src/ATen/native/Indexing.cpp

```diff
@@ -314,6 +314,46 @@ Tensor index_copy(const Tensor & self, int64_t dim, const Tensor & index, const
   return self.clone().index_copy_(dim, index, source);
 }
 
+
+Tensor& index_add__cpu(Tensor & self, int64_t dim, const Tensor & index, const Tensor & source) {
+  dim = maybe_wrap_dim(dim, self.dim());
+
+  auto numel = index.numel();
+  TORCH_CHECK_INDEX(index.dim() <= 1, "index_add_(): Index is supposed to be a vector");
+  TORCH_CHECK(index.scalar_type() == ScalarType::Long, "index_add_(): Expected LongTensor for index");
+  TORCH_CHECK(self.scalar_type() == source.scalar_type(),
+              "index_add_(): self and source must have the same scalar type");
+
+  auto index_contig = index.contiguous();
+  auto index_data = index_contig.data_ptr<int64_t>();
+
+  if (self.dim() > 1) {
+    for (auto i=0; i<numel; i++) {
```

Comment:
nit: be consistent about spacing around operators, just add a couple of spaces here.